src/components/HackEvents/HackList/index.tsx

```diff
@@ -7,14 +7,14 @@ export type EventListProps = {
   events: Array<EventObject>,
   type: string
 }
-const EventList = ({ events, type }: EventListProps) => {
+const HackEventList = ({ events, type }: EventListProps) => {
   return (
     <div className="EventList">
       {events.map((event) => {
-        return <HackEventCard event={event} type={type} />
+        return <HackEventCard event={event} type={type} key={event.uuid} />
```

Comment:
Can we increase the size of these event cards to be `width: 320px`. On some event banners, the card isn't large enough to fit the image
![image](https://user-images.githubusercontent.com/8893880/96529296-a411d500-1239-11eb-8d3f-8e14042dd2d2.png)